udata/core/spatial/commands.py

```diff
@@ -29,16 +28,14 @@
 @m.command
 def load(filename, drop=False):
     '''Load a GeoZones Bundle'''
-    tmp = tempfile.mkdtemp()
-
     if filename.startswith('http'):
         log.info('Downloading GeoZones bundle: %s', filename)
-        filename, _ = urlretrieve(filename, join(tmp, 'geozones.tar.xz'))
+        filename, _ = urlretrieve(filename, join(tmp.root, 'geozones.tar.xz'))
```

Comment:
I think you can write `tmp.path('geozones.tar.xz')` instead of manually using `join()`